torch/_decomp/decompositions.py

```diff
@@ -1067,6 +1067,37 @@ def embedding_dense_backward(
     return grad_weight.index_put([indices], grad, accumulate=True)
 
 
+@register_decomposition(aten.embedding_renorm_)
+def embedding_renorm_(
+    self: Tensor,
+    indices: Tensor,
+    max_norm: float,
+    norm_type: float,
+) -> Tensor:
+    # TODO: Assert not ported over yet
+    #   checkDim("embedding_renorm_", self_arg, 2);
+    #   checkScalarTypes("embedding_renorm_", indices_arg, {kLong, kInt});
```

Comment:
Did the same what's done in `embedding` decomp. Unified checks are introduced in the next PR in this stack.